lib/privnote-cli.js

```diff
@@ -15,14 +15,15 @@ exports.crypt = function(note, fn) {
       "has_manual_pass": false,
       "duration_hours" : 0,
       "data_type": "T",
+      "dont_ask": !opts.ask,
       "notify_email": null,
       "notify_ref" : null
     });
 
     var post_options = {
         host: "privnote.com",
         port: 443,
-        path: "/",
+        path: "/legacy/",
```

Comment:
not sure what they've got working under the hood, but this is the endpoint that's live as of now. just trying to be polite and emulate their website behavior as much as possible.